src/lib/api/submodel-registry-service/ISubmodelRegistryServiceApi.ts

```diff
@@ -1,15 +1,16 @@
 import { SubmodelDescriptor } from 'lib/types/registryServiceTypes';
```

Comment:
can you adapt the name again so it matches all other interfaces?

Reply:
the default interface naming is now:
the file with only interfaces is called with a postfix `Interface.ts`
the interface class is named with a prefix `I`